explainability-service/src/main/java/org/kie/trustyai/service/payloads/PayloadConverter.java

```diff
@@ -5,14 +5,16 @@
 import org.kie.trustyai.service.payloads.values.DataType;
 import org.kie.trustyai.service.payloads.values.TypedValue;
 
+import com.fasterxml.jackson.databind.JsonNode;
+
 import static org.kie.trustyai.service.payloads.values.DataType.*;
 
 public class PayloadConverter {
     private PayloadConverter() {
     }
 
     public static Value convertToValue(TypedValue node) {
-        final DataType type = node.getType();
+        DataType type = node.getType();
```

Comment:
I'm fine with the change but I don't see the reason, as far as I see `type` is not overridden in the method